### DISCUSSION TITLE
Discover Weblate

### DISCUSSION THREAD

**Discussion by orangesunny** [WeblateOrg/weblate#5755] [Ideas]
This is a discussion to talk about the new Weblate feature introduced in 4.5.2

Browse currently listed servers: https://weblate.org/discover/

Read the article about: https://weblate.org/news/archive/discover-your-favorite-project/

Documentation: https://docs.weblate.org/en/latest/admin/support.html#discover-weblate

**Comment by nijel** [comment]
Now it's time to drop [wiki](https://github.com/WeblateOrg/weblate/wiki) where the single page is [list of Weblate servers](https://github.com/WeblateOrg/weblate/wiki/References). I've locked it down for now - no new sites should be added there. I will keep it around for some time, please let the communities know about the move.

**Reply by tomkolp** [reply]
How can I add a clickable link in the description, e.g. to a discord?

**Reply by orangesunny** [reply]
This feature is not there and won’t be. `Discover` should enable users to find the Weblate server they long for, not all the information. We wouldn’t be able to manage the external links on a large scale.

Once the user is on your server, you have multiple ways (like announcements, info page of the projects, and design of the server) to inform them where to follow next to reach the wiki, Discord, Discourse, or anything else. I hope it’s enough :)

**Reply by comradekingu** [reply]
@nijel How do I know which communities I and others have contacted, and which images have been made, if you locked the wiki page? Cross-referencing the lists just becomes difficult.

The wiki is still a tool for contributors, because it shows which projects have CLAs, and which ones are proprietary. I have no use for a list where there isn't a distinction, and I am pretty sure that isn't what the person with the suggestion asked for either.

An editable wiki could be a great resource to have to work on ideas for meetings, and drafts for the documentation, seeing as it is near impossible to get anything merged quickly. Setting up shared repos is also inconvenient. etc. etc.

And if blogposts were put there, it would be easy to avoid stuff like:

>Weblate’s popularity is growing fast, and not all the projects open for translation contributors are on Hosted Weblate.

>Some projects are too big to not having their own Weblate instance, some want to have all on their HW, and others just like the choice of deployment without asking anyone.

>Over the time, 

>many translators have been asking Weblate support about where to translate which project. We discovered multiple servers thanks to that
(Is there a sentence missing here?)

>You can find there your favorite project

and so on.

**Reply by orangesunny** [reply]
@comradekingu I will send a note about a new feature to all of them, already did with a few. It’s up to them if they want to get listed on Discover or not. You don’t have to care about it, and it doesn’t need to be tracked on the wiki.

**Reply by comradekingu** [reply]
So you are in early beta, and don't have the required info even for current listings. Looks good though.
To me that works out to be a show-piece for Weblate itself (what I proposed), not something power-users have a need for (what I put time into making). Best of luck closing the gap though.

I hope it is somewhere prominent for admins to see in 5.2. There are a lot of instances I haven't listed, and hothereto unknown ones.

Sites need a version they largely haven't upgraded to yet to participate (if they even want to) and contributors can't help matters.
They are however asked to "tell your admin.", which isn't about to be popular beyond one person doing it, and you are already doing it for all of them it seems…
Instead they might see a blogpost full of errors, pointing to a section of the documentation that [skipped review.](https://github.com/WeblateOrg/weblate/pull/5761/files). What is the point of telling people to join "team review", if it looks like team review sends this kind of stuff?

I do care about what sort of reputation Weblate has, because it is the only viable libre translation platform, with other options being too new or not in active development.

Why any of this warrants closing, and wanting to remove the entire wiki I don't know.
I cloned it since it is something I rely on to use Weblate. You don't have to care about it, I guess.

**Reply by nijel** [reply]
> I hope it is somewhere prominent for admins to see in 5.2. There are a lot of instances I haven't listed, and hothereto unknown ones.

It's currently not, we might promote it more in the 4.6 release once we get more feedback.

> The wiki is still a tool for contributors, because it shows which projects have CLAs, and which ones are proprietary.

You are pretty much the only one who edited the content and I wouldn't be surprised if you were also the only one to use that page. It was too hidden to be found by a random contributor.

**Reply by orangesunny** [reply]
@comradekingu You’re welcome to share your points of view, just do it kindly. Please stop running around Weblate repositories being sour and rude.

**Reply by comradekingu** [reply]
@orangesunny You are welcome to share your points of view about my points of view.
>This is a discussion to talk about the new Weblate feature introduced in 4.5.2

By your own admission, this isn't the place. In my world pointing out errors and sending patches is a welcomed change.
I guess the blog, the documentation, the website, and the PRs of others is your place to run changes without intervention.
Since I am on the review team I think I should be at least able to ask why it is called "review" with that in mind, considering what I attemptedly try to fix.

@nijel Only 3 people edited since 2020. I did most anything that needed doing, so a low estimate at that. Fairly updated though.
My point is power-users. If it is the case that nobody else sees it, and thus aren't using it, fine. I just lose the history of when I found things.
If it is the case that people are using it, it should in the very least not be removed early, before the Discovery index has a chance of getting populated. I don't know what the strategy is.
I guess "discussions" could be used for my other proposed uses, but I have little hope it will.

Edit: Got kicked out of "Webkate reviewers" and the Weblate reviewers just now. A discovery to be made there.

Edit2: And then this post got flagged "off-topic".

**Reply by orangesunny** [reply]
The wiki list will stay up in the current form until Discover is filled with instances and out of the testing phase.

Yes, we have removed you from the Reviewers team as you are not helping but running around sour while creating more damage than added value. If you want to talk about it, I send you a message. I am sure we can find a way to cooperate. I agree with you that my previous message is _off-topic_. I will also mark this message _off-topic_ to give you a reaction to your writings but not interrupt other users.

**Comment by NamelessGO** [comment]
Discourse has provided their own https://discover.discourse.org/ and looks amazing with various instances in one place

In Weblate there is only a few, although both are opt-in features to join Discover, there could be improvements to get more Instances in https://weblate.org/discover/

1. For starters, the process of adding an instance to discover could be simplified, Discourse is one button, while Weblate requires one form to register. The server image, description, and URL could be taken automatically, and if it is empty, then could display a warning asking to fill before proceeding. (This could remind instances to set their own favicon as well, although i think this deserves another topic)

Weblate instances favicon
<img width="151" alt="image" src="https://github.com/WeblateOrg/weblate/assets/66227691/a73719a4-523c-4e31-a2f4-a51c19feb4a8">

Discourse instance favicon
<img width="151" alt="image" src="https://github.com/WeblateOrg/weblate/assets/66227691/d0a92915-ac6d-4074-85f5-6472197fe4af">

2. Does Weblate send admins an email regularly for newer releases? If yes, then could add a message asking admins to add their instance to Discover regularly among others tips and tricks

3. (See 1st image) "Your Weblate is not listed on weblate.org" and "Browse discovery" are too far apart, could weblate.org redirect users to Discover instead -> "Your Weblate is not listed on [Discover Weblate](https://weblate.org/discover/)"

Weblate opt-in Discover feature 
![image](https://github.com/WeblateOrg/weblate/assets/66227691/d122524f-c366-4ca9-9e9d-88dfb63b1203)

Discover Weblate form
<img width="834" alt="image" src="https://github.com/WeblateOrg/weblate/assets/66227691/1aeedc4a-7a6b-497d-92d4-536eae091f73">

Discourse opt in Discover feature
![image](https://github.com/WeblateOrg/weblate/assets/66227691/da79309b-a693-4f44-b9ba-df4fbbf57019)

Discourse rules for adding an instance to Discover
<img width="488" alt="image" src="https://github.com/WeblateOrg/weblate/assets/66227691/a25958aa-d713-4556-9339-ca3a80b0ad2a">

Discourse Discover landing page (and how to access it)
<img width="831" alt="image" src="https://github.com/WeblateOrg/weblate/assets/66227691/4f15c054-83ef-4065-b0a3-935043f814d8">
<img width="1016" alt="image" src="https://github.com/WeblateOrg/weblate/assets/66227691/c06f0b09-02c0-4125-a697-617c074a172f">
